tests/access_test.go

```diff
@@ -313,10 +313,10 @@ var _ = Describe("[rfe_id:500][crit:high][vendor:cnv-qe@redhat.com][level:compon
 			table.DescribeTable("should verify permissions on resources are correct the test user", func(resource string) {
 				testRights(resource, "yes")
 			},
-				table.Entry("[test_id:2894]given a vmi", "virtualmachineinstances"),
+				table.Entry("[test_id:2920]given a vmi", "virtualmachineinstances"),
 				table.Entry("[test_id:2831]given a vm", "virtualmachines"),
-				table.Entry("[test_id:2835]given a vmi preset", "virtualmachineinstancepresets"),
-				table.Entry("[test_id:2836][crit:low]given a vmi replica set", "virtualmachineinstancereplicasets"),
```

Comment:
These tests, don't exist anymore in polarion, so replace them we the relevant ones.